test/e2e/virtual/workspaces/virtual_workspace_test.go

```diff
@@ -165,6 +218,12 @@ func TestWorkspacesVirtualWorkspaces(t *testing.T) {
 				testOrgClient := server.virtualWorkspaceClients[0]
 				defaultOrgClient := server.virtualWorkspaceClients[1]
 
+				err := createOrgMemberRoleForGroup(ctx, server.rootKubeClient, server.orgClusterName, "team-1")
+				require.NoError(t, err, "failed to create root org roles")
+
+				err = createOrgMemberRoleForGroup(ctx, server.rootKubeClient, "root:default", "team-1")
+				require.NoError(t, err, "failed to create root org roles")
```

Comment:
nit: root org is not a term. Root workspace is.

Reply:
fixed in commit 56947bcb00c6b1746f053060eb32042705a133a2